src/hydrox/common/util.clj

```diff
@@ -46,3 +46,34 @@
      (-> proj
          (update-in [:source-paths] (fnil identity ["src"]))
          (update-in [:test-paths] (fnil identity ["test"]))))))
+
+(defn deps-read-project
+  "like `read-project` but for deps.edn'
+
+   (keys (deps-read-project (io/file \"example/deps.edn\")))
+   => (just [:description :license :name :source-paths :test-paths
+             :documentation :root :url :version :dependencies] :in-any-order)"
+  ([] (read-project (io/file "deps.edn")))
+  ([file]
+   (let [path (.getCanonicalPath file)
+         root  (subs path 0 (- (count path) 9))
+         url (str "github.com/parkside-securities/")
+         project-name (last (clojure.string/split root #"/"))
+         proj-map (read-string (slurp file))
+         dependencies (merge (:deps proj-map)
+                             (apply hash-map (remove nil? (map :extra-deps (vals (:aliases proj-map))))))
+         source-paths (vec (map (fnil identity ["src"]) (:paths proj-map)))
```

Comment:
When I test this in `customer-grapqhl`, I noticed `(dive)` applies the integration against generated source code and it takes long time. As we don't need to generate documents for those generates source code, please remove `generated_src` from `source-paths`. 

```
dev=> (dive)                                                                                                                                                                                                                        [372/8071]
CHANGED: stock_quote.clj
Associating: ([customer-graphql.streamers.stock-quote])
...
Associating: ([customer-graphql.resolver.stock])
CHANGED: account.clj
Associating: ([customer-graphql.resolver.account])
CHANGED: config.clj
Associating: ([customer-graphql.config])
CHANGED: sample_document.clj
CHANGED: pubsub_test.clj
CHANGED: graphql_test.clj
CHANGED: customer_e2e_test.clj
CHANGED: validators.clj
Associating: ([custom-spec.validators])
CHANGED: datamodel.clj
Associating: ([datamodel])
CHANGED: customer.clj
Associating: ([records.customer])
CHANGED: bdops_graphql.clj
...
```